chrome/content/zotero/xpcom/itemTreeView.js

```diff
@@ -260,6 +261,44 @@ Zotero.ItemTreeView.prototype.setTree = Zotero.Promise.coroutine(function* (tree
 });
 
 
+Zotero.ItemTreeView.prototype.setSortColumn = function() {
+	var colId, col, dir, currentCol, currentDir;
```

Comment:
By the end of the day, sticking to what makes sense should be the end goal. And I feel like this makes the most sense, knowing these variables can get initialized within if statements or be left undefined completely. It's sensible to have them declared there right at the top, as a reminder that they may or may not be initialized.